source/2019-12-14-countdown-to-the-new-year-ember-power-select.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year - Ember Power Select
+author: Richard Shea  
+tags: 2019, Addons, DecEmber
+alias: 2019-12-14-countdown-to-the-new-year-ember-power-select
+responsive: true
+---
+
+**This is the fourteenth in our DecEmber series- "Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 14
+
+Today's selection for '31 Days of Ember Addons' is [ember-power-select](https://emberobserver.com/addons/ember-power-select).
+
+READMORE
+
+### What It Does
+
+Ember Power Select provides a powerful select component, built from the ground up for Ember so that it works the way you expect an addon to work in Ember. Exceptionally [well-documented](https://ember-power-select.com/docs), Ember Power Select allows you to create everything from a straightforward list of text items to an asynchronous search driven select which incorporates graphical elements in the list. 
+
+The addon's website includes a [cookbook](https://ember-power-select.com/cookbook) for more unusual requirements as well as a [list of community provided addons](https://ember-power-select.com/addons) which address functionality outside of the scope of Ember Power Select.
+
+### Why I Like It
+
+I have grown to love using Ember Power Select while developing a CRUD-rich web app which benefits from many instances of select elements driven by asynchronous searches. I have made use of the [cookbook](https://ember-power-select.com/cookbook) to debounce those searches and so reduce the load on the server while maintaining a slick feel to the interface. The look of the resulting selects is fine just as they but the [clearly documented API](https://ember-power-select.com/docs/api-reference) has allowed me to tweak each use of the addon and so provide a better experience to the users.
```

Comment:
Currently our language linter on CI errors with the following:

```
source/2019-12-14-countdown-to-the-new-year-ember-power-select.md
  25:401-25:405  warning  `just` may be insensitive, try not to use it     just     retext-equality
  25:423-25:430  warning  `clearly` may be insensitive, try not to use it  clearly  retext-equality
```

Since the `just` isn't meant in a non-condescending manner here, I think we can just disable the lint check for this one and find a way to avoid the `clearly` - what do you think? This should do the trick:

```suggestion
<!--alex ignore just-->
I have grown to love using Ember Power Select while developing a CRUD-rich web app which benefits from many instances of select elements driven by asynchronous searches. I have made use of the [cookbook](https://ember-power-select.com/cookbook) to debounce those searches and so reduce the load on the server while maintaining a slick feel to the interface. The look of the resulting selects is fine just as they are, but the [well-documented API](https://ember-power-select.com/docs/api-reference) has allowed me to tweak each use of the addon and to provide a better experience to the users.
```